tei-py/python/structs.py

```
@@ -1,327 +1,203 @@
-    content:
-        Ordered list of inline nodes comprising the paragraph. At runtime this
-        is a plain ``list`` of ``dict``/``str`` values because msgspec cannot
-        materialise multiple mapping types in one union.
-    """
 
-    xml_id: str | None = msgspec.field(default=None, name="@xml:id")
-    content: list[Inline] = msgspec.field(default_factory=list, name="$value")
+class InlineHi(msgspec.Struct, tag="hi", tag_field="type", omit_defaults=True):
+    """Emphasised inline span."""
 
+    content: list["Inline"] = msgspec.field(default_factory=list)
```

Ruff (0.14.10)

Remove quotes from type annotation

Remove quotes

(UP037)